press.js

```diff
@@ -1,25 +1,26 @@
 export function pressKeyUX(pressedClass) {
-  let pressed
+  let pressedElement
+  let classes = pressedClass.split(' ');
 
-  function keyUp() {
-    if (pressed) {
-      pressed.classList.remove(...pressedClass.split(' '))
+  function handleKeyUp() {
+    if (pressedElement) {
+        pressedElement.classList.remove(classes)
     }
   }
 
-  function click(event) {
+  function handleClick(event) {
```

Comment:
Can you revert name changes to keep consistency with other files

Reply:
Sure, I did this out of habit and for clarity